doc/langref.html.in

```diff
@@ -9421,14 +9421,14 @@ fn doTheTest() !void {
       Supports {#link|Floats#} and {#link|Vectors#} of floats.
       </p>
       {#header_close#}
-      {#header_open|@fabs#}
-      <pre>{#syntax#}@fabs(value: anytype) @TypeOf(value){#endsyntax#}</pre>
+      {#header_open|@abs#}
+      <pre>{#syntax#}@abs(value: anytype) @TypeOf(value){#endsyntax#}</pre>
```

Comment:
Please document the return type for integers is unsigned, and therefore for integers there is no possible overflow or other illegal behavior.